packages/graphql-server/src/app.ts

```diff
@@ -115,7 +115,8 @@ export const resolvers = {
     ...ann.resolveInMutation(),
   },
   System: {
-    smtp: system.querySmtp
+    smtp: system.querySmtp,
+    license: system.queryLicense,
```

Comment:
Should not add this line in ce app.ts. Make sure that ce code should not include ee code.